js/engine.js

```diff
@@ -218,6 +218,11 @@ function unloadGame() {
 function generateTitleScreen()
 {
 	titleMode=(curlevel>0||curlevelTarget!==null)?1:0;
+	if (!!window.localStorage) {
```

Comment:
hmm i worry that this will throw an error even just checking like this (which I assume is why I wrapped it in a `try` previously: https://github.com/pancelor/PuzzleScript/commit/1afe1018e8637ebcbb067e149d9eeede298595d4 , but maybe I was just being cargo-culty before and this actually works fine. https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage doesn't really clarify things afaict)

Either way, I'll merge this and then maybe fix it later if it turns out that's necessary.

Thanks for the PR!! adding those `x`s to the level select screen is a huge improvement and I like the menu change too :)